build/TestShared/xunit.runner.json

```diff
@@ -1,5 +1,7 @@
 {
-  "parallelizeTestCollections": true,
+  "diagnosticMessages": true,
+  "methodDisplay": "classAndMethod",
   "parallelizeAssembly": true,
-  "longRunningTestSeconds": 1200
-}
\ No newline at end of file
+  "parallelizeTestCollections": true,
```

Comment:
This is the default, right?  I don't mind being explicit, but trying to understand the impact of this PR.

Reply:
Yes Andy `"parallelizeTestCollections": true` is by default true. I added this entry for readability.